latest_cov_build.py

```diff
@@ -24,32 +34,33 @@
     url = queue.buildUrl('getLatestArtifact', taskId, 'public/build/{}'.format(name))
     urlretrieve(url, os.path.join('tools', name))
 
-# Extracting coverage build artifact
-with tarfile.open('tools/target.tar.bz2', 'r:bz2') as tar:
-    tar.extractall(path='tools')
-os.remove('tools/target.tar.bz2')
-
 # Geckodriver download
 # OS information for correct geckodriver version
 bitness = platform.architecture()[0]
-latest_version = requests.get('https://api.github.com/repos/mozilla/geckodriver/releases/latest')
-data = latest_version.json()
-tag_name = data['tag_name']
-base_url = 'https://github.com/mozilla/geckodriver/releases/download/' + tag_name + '/geckodriver-' + tag_name + '-'
+
+gecko_url = get_url('https://api.github.com/repos/mozilla/geckodriver/releases/latest',
+                    'https://github.com/mozilla/geckodriver/releases/download/', 1)
+
+# Latest grcov version
+grcov_url = get_url('https://api.github.com/repos/marco-c/grcov/releases/latest',
+                    'https://github.com/marco-c/grcov/releases/download/', 0)
 
 # Linux OS
 if sys.platform.startswith('linux'):
+    grcov_url += '/grcov-linux-x86_64.tar.bz2'
```

Comment:
Move all the geckodriver and grcov download code in the same place:
```
geckodriver_url = ...
download geckodriver

grcov_url = ...
download grcov
```

Otherwise, mixing them gets confusing.